pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/loader/IndexLoadingConfig.java

```diff
@@ -289,6 +293,10 @@ public int getRangeIndexVersion() {
     return _rangeIndexVersion;
   }
 
+  public FSTType getFstIndexType() {
```

Comment:
Suggest renaming to `getFSTIndexType` to be consistent with `getFSTIndexColumns`. Same for the setter

Reply:
^^